opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -98,12 +98,14 @@ message ScopeMetrics {
 //   fields.
 //
 //     Metric
-//  +------------+
-//  |name        |
-//  |description |
-//  |unit        |     +------------------------------------+
-//  |data        |---> |Gauge, Sum, Histogram, Summary, ... |
-//  +------------+     +------------------------------------+
+//  +--------------------+
+//  |name                |
+//  |resource attributes |
+//  |scope               |
```

Comment:
I think his is misleading. Resource and Scope are in different messages, they are not part of the Metric message.

Reply:
Hm, right, although then I am not sure what's the best way to visualise it as currently we only have a diagram for Metric, and other messages are missing as well. Maybe just visualising all messages at the top of the proto?

Reply:
I'd suggest visualizing more or leaving things as-is.

Metric hierarchy is quite important to the DataModel, and it's a nuance *on top of* the:  `Resource{Signal}->Scope{Signal}->{Signal}s` hierarchy of the rest of the protocol.  Particularly because with metrics:

- We have two identities:  Metric Identity and *time series* identity.
- We are reporting aggregated values (accumulated within start/stop window) that have known aggregations for handling them.

Reply:
Yes, now I am thinking maybe introducing a tree to describe these relations. Something like:
```console
MetricsData
├── ResourceMetrics
│ ├── Resource
│ ├── SchemaURL
│ └── ScopeMetrics
│    ├── InstrumentationScope
│    ├── SchemaURL
│    └── Metric
│       ├── Name
│       ├── Description
│       ├── Unit
│       └── data
│          ├── Gauge
│          ├── Sum
│          ├── Histogram
│          ├── ExponentialHistogram
│          └── Summary
└── ...
```

Reply:
Thanks @bogdandrutu, I will fix it! Any thoughts on including something like that?

Reply:
I think your proposal helps understand, will also be good to apply to the other signals to be consistent.

Reply:
I pushed a commit, @bogdandrutu, @jsuereth, @tigrannajaryan would you mind checking it?